backend/api/tests/views/test_views_algo.py

```diff
@@ -44,11 +45,11 @@ def setUp(self):
         self.logger.setLevel(logging.ERROR)
         self.url = reverse("api:algo-list")
 
-        simple_algo = factory.create_algo(category=Algo.Category.ALGO_SIMPLE, name="simple algo")
-        aggregate_algo = factory.create_algo(category=Algo.Category.ALGO_AGGREGATE, name="aggregate")
-        composite_algo = factory.create_algo(category=Algo.Category.ALGO_COMPOSITE, name="composite")
-        predict_algo = factory.create_algo(category=Algo.Category.ALGO_PREDICT, name="predict")
-        metric_algo = factory.create_algo(category=Algo.Category.ALGO_METRIC, name="metric")
+        simple_algo = factory.create_algo(category=AlgoCategory.simple, name="simple algo")
```

Comment:
You need to rebase, this has been handled in https://github.com/Substra/substra-backend/pull/471